synfig-studio/src/gui/app.cpp

```diff
@@ -1961,6 +1977,105 @@ App::set_time_format(synfig::Time::Format x)
 	_App_time_format=x;
 }
 
+/*
+
+Save backup file without altering the current file
+
+*/
+
+bool
+App::save_backup()
+{
+	int max_backups = get_num_backup_files();
+
+	if (max_backups == 0){
+		return true;
+	}
+
+	CanvasView::LooseHandle canvas_view = App::get_selected_canvas_view();
+
+	// no canvas, return immediately
+	if (canvas_view.empty()){
+		return true;
+	}
+
+	FileSystemTemporary::Handle temporary_filesystem = FileSystemTemporary::Handle::cast_dynamic(canvas_view->get_canvas()->get_file_system());
+	// get original filename
+	String filename = temporary_filesystem->get_meta("filename");
+	String as = temporary_filesystem->get_meta("as");
+	String truncate = temporary_filesystem->get_meta("truncate");
+	if (filename.empty() || as.empty() || truncate.empty())
+		throw (String)strprintf(_("Original filename was not set in temporary container \n\n"));
+	FileContainerZip::file_size_t truncate_storage_size = stoll(truncate);
+	filesystem::Path proj_dir = filesystem::Path::dirname(filename);
+
+	// is new file, do not backup
+	if(proj_dir.u8string() == "."){
```

Comment:
That's true, I haven't thought about that. 

Maybe instead of creating a backup directory, the user can designate the backup directory in the preference list. This would allow users to save new files with no directories, however the backed up files would be saved as "Synfig 1_1.sif" and would not be renamed after saving the new file to a different name. 

@pgilfernandez, as the original author of the issue, is this what you envisioned for this feature or did you want something else?